### ISSUE TITLE
FF admin should automatically have access to all teams

### ISSUE THREAD

**Issue by SynoUser-NL** [FlowFuse/flowfuse#4086]
### Description

a FF admin should automatically have access to all teams, not only if that admin is a member of the team. Or at least be able to make itself a member of any team, no questions\acceptance asked.

### Which customers would this be available to

Enterprise Tier Only (EE)

### Have you provided an initial effort estimate for this issue?

I am not a FlowFuse team member

**Comment by SynoUser-NL** [comment]
Hi @knolleary 

I see.. that is basically what I'm seeing in my other report (https://github.com/FlowFuse/flowfuse/issues/4084). One difference is my account is Owner member of all teams and I can **not** see those in the Team dropdown. And a few other things..

I'm currently unable to try what you described in the post above, but would I be able to make myself (as admin) Owner of a team I am not Owner of?

**Comment by joepavitt** [comment]
> I'm currently unable to try what you described in the post above, but would I be able to make myself (as admin) Owner of a team I am not Owner of?

Yes, you can go to the "Members" tab of the relevant team, and invite your own user into that team with the relevant privileges.
